numba/core/types/functions.py

```diff
@@ -405,7 +405,6 @@ def get_call_type(self, context, args, kws):
         literal_e = None
         nonliteral_e = None
         out = None
-
```

Comment:
Please revert, this is a stylistic change, thanks.

Reply:
Done.

Reply:
Thanks